src/components/MenuItem/styles.module.css

```diff
@@ -15,4 +16,17 @@
     height: 208px;
     border: solid 1px borderDarkGray;
     background-color: lightGray;
+=======
```

Comment:
This looks like it's left over from a resolved merge-conflict, should be removed.